src/HtmlBuilder.php

```diff
@@ -220,9 +220,9 @@ public function linkSecureAsset($url, $title = null, $attributes = [])
      *
      * @return \Illuminate\Support\HtmlString
      */
-    public function linkRoute($name, $title = null, $parameters = [], $attributes = [])
+    public function linkRoute($name, $title = null, $parameters = [], $attributes = [] , $escape = true)
     {
-        return $this->link($this->url->route($name, $parameters), $title, $attributes);
+        return $this->link($this->url->route($name, $parameters), $title, $attributes , null , $escape);
```

Comment:
Let's add in the option for secure too - I don't see any reason to not have that option.